lms/djangoapps/commerce/views.py

```diff
@@ -123,3 +127,8 @@ def post(self, request, *args, **kwargs):  # pylint: disable=unused-argument
         except ApiError as err:
             # The API will handle logging of the error.
             return InternalRequestErrorResponse(err.message)
+
+
+def checkout_cancel(_request):
```

Comment:
Since this is essentially a static page, is it worth adding a decorator to cache the response?